package.json

```diff
@@ -27,22 +29,24 @@
   "homepage": "https://github.com/AmplicaLabs/frequency-scenario-template#readme",
   "dependencies": {
     "@dsnp/frequency-schemas": "^1.1.0",
-    "@dsnp/graph-sdk": "^1.0.3",
-    "@frequency-chain/api-augment": "^1.11.1",
-    "@polkadot/api": "^10.12.4",
+    "@dsnp/graph-sdk": "^1.1.3",
+    "@frequency-chain/api-augment": "^1.12.0",
+    "@polkadot/api": "^11.2.1",
     "@polkadot/keyring": "^12.6.2",
-    "@polkadot/types": "^10.12.4",
+    "@polkadot/types": "^11.2.1",
```

Comment:
Note: I was having trouble between account-service, siwf, and SAT and these versions. At the time, I kept the versions in line with main-net. I will do some testing later today/tomorrow and see if there are any issues.